ports/esp32/mpconfigport.h

```diff
@@ -200,6 +200,63 @@
 
 #define MP_STATE_PORT MP_STATE_VM
 
+#if CONFIG_USB_OTG_SUPPORTED
+#ifndef MICROPY_HW_ENABLE_USBDEV
+#define MICROPY_HW_ENABLE_USBDEV            (1)
```

Comment:
This is a nitpick, but you can do

```suggestion
#define MICROPY_HW_ENABLE_USBDEV            (SOC_USB_OTG_SUPPORTED)
```
... and avoid the outer `#if CONFIG_USB_OTG_SUPPORTED` statement

(The existence of `CONFIG_USB_OTG_SUPPORTED` in ESP-IDF is really only a kluge so the config system in ESP-IDF can know the value of `SOC_USB_OTG_SUPPORTED`)